memo-app/src/components/Nav.jsx

```diff
@@ -0,0 +1,15 @@
+import { useLoginStatus } from "../contexts/useLoginStatus.js";
+
+function Nav() {
+  const { isLoggedIn, handleLogIn } = useLoginStatus();
```

Comment:
nits

このほうが自然ですかね 🤔 

```suggestion
  const { isLoggedIn, handleLogin } = useLoginStatus();
```